src/main/java/com/nike/cerberus/domain/cloudformation/TagParametersDelegate.java

```diff
@@ -16,17 +16,38 @@
 
 package com.nike.cerberus.domain.cloudformation;
 
+import com.beust.jcommander.DynamicParameter;
+import com.beust.jcommander.Parameter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.nike.cerberus.command.StackDelegate.COST_CENTER_LONG_ARG;
+import static com.nike.cerberus.command.StackDelegate.OWNER_EMAIL_LONG_ARG;
+import static com.nike.cerberus.command.StackDelegate.PARAMETER_SHORT_ARG;
+
```

Comment:
Do these constants we're static importing actually belong in this class?  Would make sense to me though I'm not sure of full context.